code/modules/uplink/uplink_items.dm

```diff
@@ -918,6 +926,42 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	purchasable_from = ~UPLINK_CLOWN_OPS
 	illegal_tech = FALSE
 
+/datum/uplink_item/ammo/c38
+	name = ".38-special Speed Loader"
+	desc = "A standard issue .38-special speed loader, for use with the Detective's revolver or 'Infiltrator' coat pistol."
+	item = /obj/item/ammo_box/c38
+	cost = 1
+	purchasable_from = ~UPLINK_CLOWN_OPS
+	illegal_tech = FALSE
+
+/datum/uplink_item/ammo/c38blister
+	name = ".38-special 'Blister' Speed Loader"
+	desc = "For when you can't deside between a coat pistol and a dart pistol! These 6 cartridges can \
+			be injected with up to 10 units of your favorite poison for remote application via sidearm."
+	item = /obj/item/ammo_box/c38/dart
+	cost = 2
+	purchasable_from = ~UPLINK_CLOWN_OPS
+	illegal_tech = FALSE
+
+/datum/uplink_item/ammo/c38dumdum
+	name = ".38-special DumDum Speed Loader"
+	desc = "6 specialized fragmenting .38-special catridges, excellent for dispatching unarmored targets. \
+			Shrapnel can embed within the victim and provide a debilitating effect. Not advised for use \
+			against armored targets."
+	item = /obj/item/ammo_box/c38/dumdum
+	cost = 2
+	purchasable_from = ~UPLINK_CLOWN_OPS
+	illegal_tech = FALSE
+
+/datum/uplink_item/ammo/c38icyhot
+	name = ".38-special 'Thermal' Speed Loader 2-Pack"
+	desc = "A 2-pack of .38-special speed loaders, offering 6 Iceblox and 6 Hot Shot cartridges. Set your \
+	targets alight or freeze them in place with these specialized ammo types."
+	item = /obj/item/storage/box/syndie_kit/c38icyhot
+	cost = 4
```

Comment:
4 TC seems pretty expensive for these, they aren't *that* good in reality and can be printed from security

Reply:
Seperated out into each ammo type and made 1 TC.